templates/dockerhub/test/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM debian AS build
```

Comment:
What is this file for?

Reply:
Splitting docker image, that way the final image is smaller and without security risk.